integrations/crewai/requirements.txt

```diff
@@ -0,0 +1,4 @@
+crewai
+crewai_tools
+langchain_community
+python-dotenv
```

Comment:
Can you add versions?

Reply:
thanks @snova-luiss I've added the versions. Re the rendering of the flow thats a vscode issue, if you open in jupyter normally it works.